src/Utils.vala

```diff
@@ -81,4 +81,11 @@ public class Tardis.Utils {
 
         return new_list;
     }
+
+    public static string display_name (GLib.Volume volume) {
+        var volume_name = volume.get_name ();
+        var drive_name = volume.get_drive ().get_name ();
```

Comment:
`get_drive` returns a `Drive?` so this should probably handle the null case.

Either set a reasonable default or just return `volume_name` if it's null.

Reply:
Fixed but I've squashed it into the wrong commit sorry - one moment and I'll sort it out

Reply:
OK I think I've got it